### DISCUSSION TITLE
Translate dialog buttons

### DISCUSSION THREAD

**Discussion by PatriaMinas** [pawelsalawa/sqlitestudio#4610] [General]
If possible, please make the buttons available for portuguese translation on Crowdin. See the image here.

![botton](https://user-images.githubusercontent.com/95147314/206796917-be59879f-c223-48b4-9fc5-58067ba96635.png)

**Comment by pawelsalawa** [comment]
These come from core Qt library. It means the are not translated in the Qt project in the version currently used (5.15.2).
